### ISSUE TITLE
List of actions from call

### ISSUE THREAD

**Issue by ValWood** [pombase/website#2255]
PRINTS, only report start and end in table

PRINTs, display features as con-joined in viewer

Move domains /families to the top and other features to the bottom

Check IDs are included in pop-ups

Put Pfam at the top one per row and label thre domains on the right-hand side

Future, we need to make a decision whether to include or exclude unintegrated families

Check if Pfan-N and Pfam-B are available for import

Also, not on the call, but I noticed that the MOBIDB said "provided by InterPRO". Probably this should say "Imported from InterPro" since that is where we get them from, but InterPro  do not  create  them.Maybe in these cases we should be more explicit ->  
e.g Provided by MOBIDB  via InterPRO?

**Comment by kimrutherford** [comment]
> MOBIDB said "provided by InterPRO".

Could you add a screenshot?  I can't find that.

**Comment by kimrutherford** [comment]
See also:
  - https://github.com/pombase/website/issues/2203#issuecomment-2385619671

**Comment by ValWood** [comment]
I think I imagined it!

But we should change this to say "Imported from" 
<img width="504" alt="Screenshot 2024-10-16 at 11 31 11" src="https://github.com/user-attachments/assets/adc6edfd-2e53-4682-aa10-fdde5251fae3">

**Comment by kimrutherford** [comment]
> But we should change this to say "Imported from"

OK, I've changed that on my desktop.

**Comment by kimrutherford** [comment]
> PRINTs, display features as con-joined in viewer

Should we do that?  InterPro has them as them as separate features:
  https://www.ebi.ac.uk/interpro/protein/reviewed/Q9Y7Z8/

**Comment by ValWood** [comment]
I think we shoudn't follow interPro on this one. Its  clearer as  single conjoined feature.
@PCarme  do you agree?

**Comment by PCarme** [comment]
Yes, in term of visualisation I think it makes more sense as a single feature

**Comment by kimrutherford** [comment]
> Check if Pfan-N and Pfam-B are available for import

The InterProScan output gives no hint about this.  The InterPro website only mentions Pfam-A:
https://www.ebi.ac.uk/interpro/download/pfam/

There is a Pfam-B file on the FTP site:
https://ftp.ebi.ac.uk/pub/databases/Pfam/current_release/

This old blog post says that Pfam-B is not included in the main Pfam website so maybe it's not included in InterPro:
https://xfam.wordpress.com/2020/06/30/a-new-pfam-b-is-released/

**Comment by ValWood** [comment]
Pfam-b not urgent.  but useful if we can link it to the interpro to see other 'members'

**Comment by kimrutherford** [comment]
Note to self: fix JaponicusDB config

**Comment by kimrutherford** [comment]
> PRINTS, only report start and end in table

I've implemented that now.  It's a configuration setting per feature type so we can do the same for other rows in the table if needed.

**Comment by kimrutherford** [comment]
Next thing to fix:

> Check IDs are included in pop-ups

**Comment by kimrutherford** [comment]
> > Next thing to fix:
>
> Check IDs are included in pop-ups

Done!

I'm going to fix the JaponicusDB configuration next and then I think we can close this issue.

**Comment by kimrutherford** [comment]
> I'm going to fix the JaponicusDB configuration next

That's done.  I'll push the changes to GitHub once all the other changes are released.
